### PR TITLE
Miscellaneous changes on the fly

### PR DESCRIPTION
**Pull request by Volte6**
* Added more info to `README.md`
* Removed a lot of commented out code (It will be retained in git history)
* Added some comments, cleaned up some whitespace
*  Added a check for JS variable existence

### CODE REVIEW

File: creator.php [hunk 1]
```diff
@@ -5,13 +5,9 @@
 include "incs/header.inc.php";
 
 $c = new common();
-//$s = new Story();
```
Comment (taylorbayouth) [inline creator.php:8]: Leave this here

Comment (Volte6) [reply creator.php:8]: Will put back in

File: creator.php [hunk 2]
```diff
@@ -22,12 +18,16 @@
 
 <script src="js/treant/collapsable.js"></script>
 <script>
-    tree = new Treant( chart_config );
-</script>
+	if ( typeof(chart_config) != "undefined" ) {
```
Comment (taylorbayouth) [inline creator.php:21]: This looks good

File: creator.php [hunk 3]
```diff
@@ -22,12 +18,16 @@
...
 
+    	tree = new Treant( chart_config );
```
Comment (taylorbayouth) [inline creator.php:23]: Not sure here

File: get-json.php [hunk 4]
```diff
@@ -15,34 +15,5 @@
 // Send the header saying this will ONLY be json displaying
 header('Content-Type: application/json');
 
-/*
```
Comment (taylorbayouth) [inline get-json.php:18]: Testing

**Review by taylorbayouth — CHANGES_REQUESTED** [review]
See comments below

File: creator.php (hunk 3 above)
Comment (Volte6) [reply creator.php:23]: Sorry

**Review by taylorbayouth — APPROVED** [review]

**Pull request merged**